docs/api.adoc

```diff
@@ -0,0 +1,60 @@
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+ifndef::rootdir[:rootdir: ../]
+:imagesdir: {rootdir}/images
+:toc:
+
+= JSON RPC API
+
+JSON-RPC is a remote procedure call protocol encoded in JSON. You must call the API with a POST HTTP request.
+
+We use the link:http://www.jsonrpc.org/specification[version 2 of the protocol].
+
+== emerald_currentVersion
+
+Returns the current version.
+
+Arguments: none
+
+Result:
+
+    * version - the current version
+
+== emerald_newAccount(name, desc, passphrase)
+
+Creates new keystore file and caches it.
+
+Arguments:
+
+    * name - name for account (optional)
```

Comment:
Can we put optional parameters in the end? It seems like JSON RPC uses an array to represent all parameters, if I understand this correct.

Putting optional ones to the end would avoid unnecessary confusion and potential code failures (for example, get an account description as the passphrase).